src/components/OrganisationPublicView/JobsTab/index.tsx

```diff
@@ -0,0 +1,32 @@
+import useFetchAllOrgJobs from 'hooks/useFetchAllOrgJobs';
+import { useNavigate, useParams } from 'react-router-dom';
+import JobCards from 'components/Jobs/JobCards';
+import styles from './index.module.scss';
+
+const JobsTab = () => {
+  const { orgId } = useParams();
+  const navigate = useNavigate();
+  const { data, isLoading } = useFetchAllOrgJobs(orgId as unknown as number);
```

Comment:
If you change the type to `string | undefined` in hooks file then no need to write `as unknown as number`